pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -604,6 +604,11 @@ in with passthru; stdenv.mkDerivation (finalAttrs: {
       inherit src;
       name = "python${pythonVersion}-${version}-doc";
 
+      patches = optionals (pythonAtLeast "3.9" && pythonOlder "3.10") [
+        # https://github.com/python/cpython/issues/98366
+        ./3.9/fix-sphinx-gettext.patch
```

Comment:
`fetchpatch` can be used instead of commiting patch to nixpkgs

https://github.com/python/cpython/pull/99358/commits/192a19b0fd179a41c24cacde364b818a5988063a

Reply:
Ah, I had thought the restriction on using `fetchpatch` applied to anything in the file, but it makes sense that it wouldn't apply for this derivation, since it's in `passthru`.

Fixed.